docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -316,6 +325,7 @@ available amount is simultaneously allocated to Pods.
 **Note:** Opaque integer resources are Alpha in Kubernetes version 1.5.
```

Comment:
I think we can drop this note entirely now.

Reply:
agreed